tests/utils/getScenarioContent.test.tsx

```diff
@@ -74,7 +75,7 @@ describe('The determination interview is not yet scheduled (scenario 1) when all
     const pendingDetermination = getMockPendingDetermination()
     pendingDetermination.determinationStatus = null
     pendingDetermination.scheduleDate = null
-    pendingDetermination.requestDate = 'any non-null value'
+    pendingDetermination.requestDate = formatFromApiGateway(-30)
```

Comment:
Note: requestDate must now be a valid date, so these tests pick a random date in the past.